internal/pkg/esboot/ilm.go

```diff
@@ -0,0 +1,226 @@
+package esboot
+
+import (
+	"context"
+	"encoding/json"
+	"fleet/internal/pkg/esutil"
+	"io"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/elastic/go-elasticsearch/v8"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+)
+
+// Can be cleaner but it's temporary bootstrap until it's moved to the elasticseach system index plugin
+
+const (
+	defaultILMPolicy = `
+	{
+		"policy":{
+		   "phases":{
+		   }
+		}
+	}`
+
+	ilmRolloverSize = 300
+	ilmRolloverAge  = 30
+	ilmDeleteAge    = 90
```

Comment:
Lets not delete data by default, at least not now. I think this could bite us later if we forget to remove it.